models/us_census_acs/code/parse_pums_dict.py

```
@@ -0,0 +1,287 @@
+            with z.open(sorted(members)[0]) as m:
+                line = io.TextIOWrapper(m, encoding="latin-1").readline()
+                cols, seen_local = [], set()
+                for c in line.rstrip("\n").split(","):
+                    u = RENAME.get(c.strip().upper(), c.strip().upper())
+                    if (
+                        u not in seen_local
+                    ):  # drop dup if both old+new present in a file
+                        cols.append(u)
+                        seen_local.add(u)
+                return cols
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)